src/Core/src/Primitives/WebProcessTerminated.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Microsoft.Maui
+{
+	public class WebProcessTerminated
```

Comment:
Maybe iOS should also have Sender for consistency?

Reply:
Added!